### ISSUE TITLE
Missing decpration warning for wrapper.contains() from documentaion

### ISSUE THREAD

**Issue by SmilingJoe** [vuejs/vue-test-utils#1582]
### What problem does this feature solve?
Recently upgraded @vue/test-utils from 1.0.0-beta.31 to 1.0.2. Upon running our unit tests, received the following error:
ERROR: '[vue-test-utils]: contains is deprecated and will be removed in the next major version Use `wrapper.find`, `wrapper.findComponent` or `wrapper.get` instead'

Noticed that there is no warning of this deprecation specified in the API Wrapper documentation:
https://vue-test-utils.vuejs.org/api/wrapper/#contains

Providing this deprecation warning, along with a specific replacement example, in the documentation will save time for developers.

### What does the proposed API look like?
Propose adding the following deprecation warning to the top of relevant API Wrapper "contains" document:
```
| Deprecation warning
| `contains` is deprecated and will be removed in future releases. Use 'wrapper.find', `wrapper.findComponent` or `wrapper.get` instead.
```

**Comment by SmilingJoe** [comment]
Just noticed that there is a deprecation warning on the 'wrapper.find' document (https://vue-test-utils.vuejs.org/api/wrapper/#find), which would obviously preclude that method from being part of this deprecation warning.

**Comment by afontcu** [comment]
Hi! You're right, we're missing a deprecation warning on docs. Fancy to open up a PR? :)

**Comment by SmilingJoe** [comment]
Sure, I can do that.

**Comment by afontcu** [comment]
Hi @SmilingJoe! Did you have the chance to take a look at this? :)

**Comment by SmilingJoe** [comment]
Sorry @afontcu I've been tied up with other projects. Looking into this now as I have some free time.

Raised PR: https://github.com/vuejs/vue-test-utils/pull/1612

**Comment by afontcu** [comment]
Fixed in #1612!
